pkg/inmemorychannel/message_dispatcher_benchmark_test.go

```diff
@@ -96,7 +91,7 @@ func BenchmarkDispatcher_dispatch_ok_through_2_channels(b *testing.B) {
 
 	dispatcher := NewMessageDispatcher(dispatcherArgs)
 	requestHandler := kncloudevents.CreateHandler(dispatcher.handler)
-	httpSender.Client = mockedHTTPClient(clientMock(channelA.URL.Host, transformations.URL.Host, channelB.URL.Host, receiver.URL.Host, requestHandler))
+	mockedHTTPClient(clientMock(channelA.URL.Host, transformations.URL.Host, channelB.URL.Host, receiver.URL.Host, requestHandler))
```

Comment:
I think this is not needed anymore, is it?
```suggestion
```

Reply:
Actually, I did try to remove  line 94 but it will create cascading failures during test runs since requestHandler at  line 93 will throw error for not being used, If line 93 is removed since it is not used 92 would fail for dispatcher not being used.. then dispatcherArgs, multiChannelFanoutHandler, config, multichannelfanout.

Reply:
ok, then lets remove the benchmark, as it is anyhow not benchmarking the dispatcher anymore.

Reply:
Yes, Removed the benchmark test file completely. Since it had only one main function and other two functions which were being used in main.